modules/products/bamboo/main.tf

```diff
@@ -26,3 +26,18 @@ module "database" {
   vpc                     = var.vpc
   db_name                 = var.db_configuration["db_name"]
 }
+
+module "nfs" {
+  source = "../../AWS/nfs"
+
+  namespace               = var.namespace
+  product                 = local.product_name
+  requests_cpu            = var.nfs_requests_cpu
+  requests_memory         = var.nfs_requests_memory
+  limits_cpu              = var.nfs_limits_cpu
+  limits_memory           = var.nfs_limits_memory
+  capacity                = var.shared_home_size
+  availability_zone       = var.eks.availability_zone
+  shared_home_snapshot_id = null
```

Comment:
Isn't this defaulted to `null`? Can we just skip it?

Reply:
We can, but having this null is more explicit and it may be more clear

Reply:
dropped this